examples/ex_ekf_vector.py

```diff
@@ -44,7 +44,7 @@
     range_freqs,
 )
 
-gt_data, input_data, meas_data = dg.generate(x0, 0, 10, noise=True)
+gt_data, input_data, meas_data = dg.generate(x0, 0, 10, noise=noise_active)
 
 # ##############################################################################
 # Run Filter
```

Comment:
Should x be corrupted with random sample drawn from P0? And be exectued if noise_active = True

Reply:
yup you're right. i'll make those fixes!